example/speech_recognition/stt_io_iter.py

```diff
@@ -86,7 +86,7 @@ def __init__(self, count, datagen, batch_size, num_label, init_states, seq_lengt
             audio_paths = audio_paths
             texts = texts
 
-        self.trainDataList = zip(durations, audio_paths, texts)
+        self.trainDataList = list(zip(durations, audio_paths, texts))
```

Comment:
can you precise what you mean here? 
```python
        self.trainDataIter = iter(self.trainDataList)
```
I think this line gets a list_iterator out of the list and that is what is used later one.

Reply:
instead of doing - 
```
self.trainDataList = list(zip(durations, audio_paths, texts))
self.trainDataIter = iter(self.trainDataList)
```
could we just do
```
self.trainDataList = zip(durations, audio_paths, texts)
```
won't it be the same thing?

Reply:
The reason why I explicitly converted it to a list was because it is used in `random.shuffle` that expects an actual list rather than an iterator.

Reply:
ah okay. makes sense.